Objects/stringlib/fastsearch.h

```diff
@@ -753,6 +753,22 @@ STRINGLIB(count_char)(const STRINGLIB_CHAR *s, Py_ssize_t n,
 }
 
 
+static inline Py_ssize_t
+STRINGLIB(count_char_no_maximum)(const STRINGLIB_CHAR *s, Py_ssize_t n,
```

Comment:
Maybe make that function private (I don't think it should be exposed except in this module).

Reply:
It is private, as it is static. The STRINGLIB macro is to prevent name clobbering. This function will be generated for STRINGLIB_CHAR==Py_UCS1, Py_UCS2 and PyUCS4. I think keeping it this way is correct. But I may be wrong of course. How do you suggest making it private?

Reply:
Oh just by adding an underscore before its name (I should have been clearer when I said "private", I meant it in the naming but I think we don't care about underscores in C files).

Reply:
No, we don't use underscore prefix in Python for static functions. Moreover, the macro adds a prefix such as `ucs1lib_`.